knowledge-base/actions/setup-java/action-security.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Setup Java JDK'
+outbound-endpoints:
+  - fqdn: coveralls.io
```

Comment:
Thanks for submitting PR!

I think in this case you have added endpoints for step 6 [here](https://app.stepsecurity.io/github/harden-runner-canary/dassana/actions/runs/1622113198). The `actions/setup-java` action is at `step 5`. That step does not show any outbound traffic likely because of caching. So in this case you will need to rely only on a code-review of https://github.com/actions/setup-java. The final `action-security.yml` will be similar to other similar actions [here](https://github.com/step-security/secure-workflows/tree/main/knowledge-base/actions). It should be downloading java from some website. It should not be making calls to coveralls.io etc.

Reply:
Thanks for pointing out the caching thing !! 

As mentioned in the readme, while using `setup-java` [click here](https://github.com/actions/setup-java#supported-distributions) we can use different java-distribution  depending on the requirement, each one of them is being fetched from different source. I have added `fqdns` of those domains.